### ISSUE TITLE
SERVER CRASH ON STARTUP

### ISSUE THREAD

**Issue by CyanideJesus** [StrangeLoopGames/EcoIssues#20646]
\
**User Id: (Not logged user)**
**Version:** 0.9.2.4 beta release-156

**Steps to Reproduce:**
1. Start server

**Expected behavior:**
Started the server

**Actual behavior:**
The following error message popped up:
```
<size=150%>Connection Failed</size>

Server encountered an exception:

<size=60.00%>Exception: NullReferenceException
Message:Object reference not set to an instance of an object.
Source:Eco.Gameplay

System.NullReferenceException: Object reference not set to an instance of an object.
   at Eco.Gameplay.Economy.Jobs.RecurringTransfer.get_ServiceMarkedUpName()
   at Eco.Core.Utils.CoreReflectionUtils.<>c__DisplayClass18_0`2.<FuncMethodHelper>b__0(TTarget target)
   at Eco.Core.Controller.SyncPropertyInfo`1.<>c__DisplayClass16_0.<.ctor>b__1(Object obj)
   at Eco.Core.Controller.SyncPropertyInfo`1.GetValue(Object obj)
   at Eco.Core.Controller.ControllerManager.PackageProperty(IMvcNetClient boundClient, IController controller, ISyncPropertyInfo propertyInfo)
Outer Exceptions:
Error during packaging property 'IncomeAndExpenses' of 'Eco.Gameplay.Players.User'
Error during packaging property 'ServiceMarkedUpName' of 'Eco.Gameplay.Economy.Jobs.RecurringTransfer'</size>
```

**Do you have mods installed? Does the issue happen when no mods are installed?:**
No mods installet

**Please provide log files. Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file

**Comment by CyanideJesus** [comment]
This has happened after I have created elected titles with wages. May the wages have anything to do with it? It's happened twice due to that.

**Comment by SlayksWood** [comment]
@CyanideJesus Thank you for the report. Can you provide save files? Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Uploading-a-world-save
You can also take a screenshot of demographic with wage to help us investigate it.

**Comment by CyanideJesus** [comment]
Here is the link. I cannot take a screenshot of demographic with wage if that demands my starting the server, as that is not possible, hence my issue.

https://drive.google.com/drive/folders/14WKvOzuJtZrihrvyWWFlz_15eTUqQEyf?usp=sharing

**Comment by IcedForge** [comment]
Unfortunately it seems the file is not publicly accessible, could you change permissions for this?

**Comment by IcedForge** [comment]
Looks like its fixed and loaded up fine for 0.9.2.5 (not yet released) ( there is an incorrect registrar title set for your elected titles ( president i assume was a registrar title, which was changed into an elected title? or vice versa ) which caused titles that had wages on them to go invalid causing all kinds of problems.

**Comment by IcedForge** [comment]
Closing this issue as it should be resolved in the next hotfix (0.9.2.5)
